src/app/components/pages/registration/registration.component.html

```diff
@@ -58,15 +58,8 @@
 
           <mat-form-field appearance="fill" class="block mb-6">
             <mat-label>Address</mat-label>
-            <input type="text"
-              placeholder="Enter/select your address"
-              aria-label="Address"
-              matInput
-              appAutocompleteForceSelection
-              formControlName="address"
-              [matAutocomplete]="autoAddress"
-              required
-            >
+            <input type="text" placeholder="Enter/select your address" aria-label="Address" matInput
```

Comment:
We can leave this for now but when an element has a bunch of properties I prefer the longer form above for readability.

Reply:
I didn't change anything here, how did this get edited?